functional-tests/README.md

```diff
@@ -1,6 +1,7 @@
 # Functional Test Automation
 
-This project automates a selection of functional tests on multiple browsers using Geb integrated with Spock and Gradle. It helps to efficiently check if the addition of new features or bug fixes has broke any previously developed features.
+This project automates a selection of functional tests on multiple browsers using Geb integrated with Spock and Gradle. It helps to efficiently check if the addition of new features or bug fixes has broke any previously developed features. The test strategy for this project is discussed on the mds confluence page:
+https://apps.nrs.gov.bc.ca/int/confluence/pages/viewpage.action?pageId=39651197
```

Comment:
Since this is a public repo, maybe add a note that this link requires IDIR login and access to confluence site?

Reply:
good idea

Reply:
done